src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesList = sourceString.replaceAll('\n', ' ').split(';');
+  const stylesObject = {};
+  const filtredStylesList = stylesList.filter((val) => val.trim().length > 0);
+
+  for (let i = 0; i < filtredStylesList.length; i++) {
+    filtredStylesList[i] = filtredStylesList[i].split(':');
```

Comment:
The code assumes that every style in `filtredStylesList` contains a colon and splits it into exactly two parts. If a style does not contain a colon or contains multiple colons, this could lead to unexpected behavior. Consider adding a check to ensure that the split results in exactly two parts before proceeding.